### DISCUSSION TITLE
Teleportation to spawn after login

### DISCUSSION THREAD

**Discussion by MrWay** [Zrips/CMI#2563] [Q&A]
**Question: **
Hey, I would like to ask if it is possible to set that player will be at his last position before disconnecting after reconnecting.

**Comment by 456dev** [comment]
```Spawn:
  # Forces players to login in defined spawn point when logging into server
  SpawnOnJoin: false
```


this is the default config

**Comment by CloudeLecaw** [comment]
you want them that they log in where they log out? then dont activate a teleport to spawn on login^^

Well all is said allready by the456gamer.

ticket can be closed
